lib/config/presets/internal/replacements.ts

```diff
@@ -3,8 +3,10 @@ import type { Preset } from '../types';
 import type { PresetTemplate, Replacement } from './auto-generate-replacements';
 import { addPresets } from './auto-generate-replacements';
 
+const { $schema, ...tempPresets } = replacementGroups;
+
 /* eslint sort-keys: ["error", "asc", {"caseSensitive": false, "natural": true}] */
-export const presets: Record<string, Preset> = replacementGroups;
+export const presets: Record<string, Preset> = tempPresets;
```

Comment:
```suggestion
export const presets: Record<string, Preset> = replacementGroups;
delete presets['$schema'];
```

what about this? so we don't need an extra variable.

Reply:
TS won't like it .. had to do the rpevious shenanigans for TS only